pkg/kwok/controllers/node_controller.go

```diff
@@ -252,6 +252,9 @@ func (c *NodeController) watchResources(ctx context.Context, opt metav1.ListOpti
 		return err
 	}
 
+	// TODO: should record last resourceVersion for next round
+	opt.AllowWatchBookmarks = true
```

Comment:
``` go
type resourceVersionGetter interface {
  GetResourceVersion() string
}
```

I'm not sure if adding this to the loop will achieve this effect?

``` go
opt.ResourceVersion = event.Object.(resourceVersionGetter).GetResourceVersion()
```

Reply:
Basically, yes. Let me handle it in the follow-up, since I am trying to use existing informer pattern.